plugins/spi/SPIOutput.cpp

```diff
@@ -375,7 +428,7 @@ void SPIOutput::IndividualLPD8806Control(const DmxBuffer &buffer) {
   if (!output)
     return;
 
-  const unsigned int length = std::min(m_pixel_count * LPD8806_SLOTS_PER_PIXEL,
+  const unsigned int length = min(m_pixel_count * LPD8806_SLOTS_PER_PIXEL,
```

Comment:
I think this wants to be back to std:: unless you've done a using.

Reply:
i deleted the using and added the namespace things like `std::` first because i misinterpreted your comment... 26c5181d683ad78c1dec2e36adab56e41bdc2db1
then re-add it. ceea93d7825470ab69ead4c47d49068f18063e21
i don't know what you are preferring (and i don't know what is 'good style' here..)

Reply:
I think the vague theory is use using if we've got a few hits, but probably not if we've only got one, as it pollutes the namespace a little, as well as making things marginally less clear. i.e. the eternal balance of being concise versus clear.

This file seems to have a "using std::auto_ptr;" and no other mention of auto_ptr, so that's a good start. But then we have vector once, but using. So we have no hard and fast rules, on the .cpp files anyway; it's more critical in the header files, although I can't remember exactly why off the top of my head.

Reply:
Sorry @s-light bump, can we remove the unused "using std::auto_ptr;" please (it's too far up the file for me to comment). I realise it's not your problem either!